test/common/quic/quic_stat_names_test.cc

```diff
@@ -0,0 +1,29 @@
+#include <string>
+
+#include "common/quic/quic_stat_names.h"
+
+#include "test/mocks/stats/mocks.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Quic {
+
+class QuicStatNamesTest : public testing::Test {
+public:
+  QuicStatNamesTest() : scope_(*symbol_table_), quic_stat_names_(*symbol_table_) {}
+
+  Stats::TestUtil::TestSymbolTable symbol_table_;
+  Stats::TestUtil::TestStore scope_;
+  QuicStatNames quic_stat_names_;
+};
+
+TEST_F(QuicStatNamesTest, QuicConnectionCloseStats) {
```

Comment:
We should doc up the new stats somewhere under docs/

Reply:
I added a new section in "Per listener stat" for Http3 stats.
I think it's a better place to document these new stats compared to codec_stats from the perspective of stats scopes.